packages/components/src/components/DetailsHeader/DetailsHeader.jsx

```diff
@@ -28,7 +28,9 @@ class DetailsHeader extends Component {
         stepStatus.terminated || {});
     }
 
-    if (!startTime || !endTime) {
+    const wrongDateChecker = new Date(startTime).getTime()
+
+    if (!startTime || !endTime || wrongDateChecker == 0) {
```

Comment:
If the startTime is either an empty string `''` or `undefined` this will not give the expected result.

`new Date('')` and `new Date(undefined)` both result in an 'Invalid Date' object, and the call to `getTime()` will return `NaN` instead of 0.

Since this is a relatively simple expression we can just inline it in the if-statement after we've already checked that `startTime` is truthy so we know it won't hit these cases. 

```suggestion
    if (!startTime || !endTime || new Date(startTime).getTime() == 0) {
```